Source/MSBuildTools.Unity/Packages/com.microsoft.msbuildforunity/Editor/ProjectGenerator/Scripts/Exporters/Extensions.cs.cs

```diff
@@ -0,0 +1,52 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
```

Comment:
nit: it looks like this file has an extra *.cs extension in its name

Reply:
Are you saying that this Extensions file that is a .cs class has an extra extension which is .cs? :D

---
In reply to: [372603757](https://github.com/microsoft/MSBuildForUnity/pull/116#discussion_r372603757) [](ancestors = 372603757)